internal/conf/configuration_test.go

```diff
@@ -15,14 +15,16 @@ func TestMain(m *testing.M) {
 
 func TestGlobal(t *testing.T) {
 	os.Setenv("GOTRUE_SITE_URL", "http://localhost:8080")
-	os.Setenv("GOTRUE_DB_DRIVER", "mysql")
+	os.Setenv("GOTRUE_DB_DRIVER", "postgres")
 	os.Setenv("GOTRUE_DB_DATABASE_URL", "fake")
 	os.Setenv("GOTRUE_OPERATOR_TOKEN", "token")
 	os.Setenv("GOTRUE_API_REQUEST_ID_HEADER", "X-Request-ID")
 	os.Setenv("GOTRUE_JWT_SECRET", "secret")
 	os.Setenv("API_EXTERNAL_URL", "http://localhost:9999")
+	os.Setenv("GOTRUE_HOOK_MFA_URI", "postgres://postgres/auth/count_failed_attempts")
```

Comment:
IMO would prefer ENABLED + URI just in case. Feels cleaner.

Reply:
Also regarding the scheme, would prefer `pg-function` or something unrelated to `postgres` as that one signals a PostgreSQL connection string, which this is not.

Reply:
Sounds good, ah yes sorry iirc we discussed the scheme but I forgot about it while writing this - will amend.

Thanks for flagging!